charts/grafana/templates/_config.tpl

```diff
@@ -81,7 +81,14 @@ download_dashboards.sh: |
 {{- range $provider, $dashboards := .Values.dashboards }}
   {{- range $key, $value := $dashboards }}
     {{- if (or (hasKey $value "gnetId") (hasKey $value "url")) }}
-  curl -skf \
+{{- if $value.url }}
```

Comment:
I guess, if someone changes the URL logic below, this may get forgotten. Not sure, if it make sense to move the  Source URL logic to a dedicated template helper.

WDYT?

Reply:
@jkroepke i assume that if the logic is changed then lines 120 and below will also be changed right ? : 
https://github.com/grafana/helm-charts/blob/c30fe288c52dcedca27d1da3cbf9e3771bb52511/charts/grafana/templates/_config.tpl#L120

not sure for now it worth it to move it into a helper template even though it would better in the future :)

Reply:
Maybe @zanhsieh has an opinion here.

Reply:
@xinity @jkroepke 
I would prefer

1. Move source url to the _helper.tpl
2. Set default `revision` as 1 as well set default source url. 
3. Move `echo "downloading: {{ $key }} dashboard"` out of `if` since both conditions output that string.